backstop/tests/menu.html

```diff
@@ -617,72 +617,87 @@ <h3>Header</h3>
-            <span class="iui-label">App name</span>
+            <span class="iui-header-brand-label">App name</span>
           </div>
 
-          <div class="iui-divider"></div>
+          <div class="iui-page-header-divider"></div>
 
-          <nav aria-label="breadcrumbs">
-            <!-- Project Button -->
-            <button
-              class="iui-button iui-borderless iui-large iui-header-button iui-header-dropdown-button"
-              type="button"
-              aria-label="Project picker"
-              aria-expanded="false"
-              aria-haspopup="true"
-              id="test-button-1"
-            >
-              <div class="iui-button-icon iui-header-button-icon demo-picture"></div>
-              <span class="iui-button-label">
-                <div>Project Alpha</div>
-                <div class="iui-description">0x0123456789</div>
-              </span>
-              <svg-caret-down-small
-                class="iui-button-icon"
-                aria-hidden="true"
-              ></svg-caret-down-small>
-            </button>
-
-            <svg-chevron-right
-              class="iui-chevron"
-              aria-hidden="true"
-            ></svg-chevron-right>
-
-            <!-- IModel Button -->
-            <button
-              class="iui-button iui-borderless iui-active iui-large iui-header-button iui-header-dropdown-button"
-              type="button"
-              aria-label="Model picker"
-              aria-expanded="true"
-              aria-haspopup="true"
-              aria-current="location"
-              id="demo-header-imodel-button"
-            >
-              <svg-model
-                class="iui-button-icon iui-header-button-icon"
-                aria-hidden="true"
-              ></svg-model>
-              <span class="iui-button-label">
-                <div>Model Beta</div>
-                <div class="iui-description">0x0123456789</div>
-              </span>
-              <svg-caret-down-small
-                class="iui-button-icon"
+          <nav
+            class="iui-header-breadcrumbs"
+            aria-label="Breadcrumbs"
+          >
+            <ol class="iui-breadcrumbs-list">
+              <li class="iui-breadcrumbs-item">
+                <span class="iui-header-breadcrumb-button-container">
+                  <button
+                    class="iui-header-breadcrumb-button"
+                    type="button"
+                    aria-label="Project picker"
+                    aria-expanded="false"
+                    id="test-button-1"
+                  >
+                    <img
+                      class="iui-header-breadcrumb-button-icon"
+                      src="../assets/image-test.jpg"
+                      draggable="false"
+                    />
+                    <span class="iui-header-breadcrumb-button-label">
+                      <div class="iui-header-breadcrumb-button-label-label">Project Alpha</div>
```

Comment:
looking at difference of images... is project alpha misaligned now (text with caret) or it was misaligned before?

Reply:
**Before:**
![Screen Shot 2022-05-19 at 3 51 40 PM](https://user-images.githubusercontent.com/849817/169392270-b910acda-2b88-418f-892e-06333e8c48d1.png)

**After:**
![Screen Shot 2022-05-19 at 3 52 19 PM](https://user-images.githubusercontent.com/849817/169392267-dca501f9-cce6-460a-bca7-6544b07fcfc0.png)